### PR TITLE
[Android] Add Simple NNStremaer example for Android

### PR DESCRIPTION
**Pull request by jijoongmoon**
# PR Description

Add Simple NNStreamer example for Android

**Changes proposed in this PR:**
- Added Android.mk
- Added nnstreamer_android.c (test tensor_converter)

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

**Comment by myungjoo** [comment]
Question: is this for a binary in Android Platform? or a binary in Android Application (.apk)?

**Comment by jijoongmoon** [comment]
Binary in Android Application. In order to build this, libnnstreamer.so and gstreamer shared libraries are required.

File: android/nnstreamer_android/nnstreamer_android.c [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ * @author  Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug     No known bugs except NYI.
+ *
+ */
+
+#include <string.h>
+#include <stdlib.h>
+#include <gst/gst.h>
+#include <tensor_common.h>
+#include <tensor_repo.h>
```
Comment (myungjoo) [inline android/nnstreamer_android/nnstreamer_android.c]: tensor_common.h and tensor_repo.h are not exported. Thus, if this is for an application, it is illegal to use them.

File: android/nnstreamer_android/nnstreamer_android.c [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+int
+main (int argc, char *argv[])
+{
+  GstElement *pipeline;
+
+  gst_init (&argc, &argv);
+
+  pipeline =
+      gst_parse_launch
+      ("videotestsrc num-buffers=1 ! video/x-raw,format=RGB,width=280,height=40,framerate=0/1 ! tee name=t ! queue ! videoconvert ! video/x-raw, format=BGRx ! tensor_converter silent=TRUE ! filesink location=\"test.bgrx.log\" sync=true t. ! queue ! filesink location=\"test.rgb.log\" sync=true",
```
Comment (leemgs) [inline android/nnstreamer_android/nnstreamer_android.c]: Looks good.

File: android/nnstreamer_android/nnstreamer_android.c [hunk 3]
```diff
@@ -0,0 +1,45 @@
+/**
+ * @file    nnstreamer_android.c
+ * @date    07 Mar 2019
+ * @brief   test case for gst_aprse_launch
```
Comment (leemgs) [inline android/nnstreamer_android/nnstreamer_android.c]: s/gst_aprse_launch/gst_parse_launch/g


And, Please depicts details on this application to test gst_parse_launch() function for developer that want to deploy nnstreamber-based application on Android device.

File: android/nnstreamer_android/nnstreamer_android.c [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+ * @see     https://github.com/nnsuite/nnstreamer
+ * @author  Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug     No known bugs except NYI.
+ *
+ */
+
+#include <string.h>
+#include <stdlib.h>
+#include <gst/gst.h>
```
Comment (leemgs) [inline android/nnstreamer_android/nnstreamer_android.c]: FYI, @jaeyun-jung starts installing Android Studio to launch a NNstreamer GUI application for Android device. :)

File: android/nnstreamer_android/Android.mk [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+bz2 harfbuzz nnstreamer
+
+ifeq ($(NO_AUDIO), false)
+BUILDING_BLOCK_LIST += gstaudio-1.0 gstbadaudio-1.0 gstaudioconvert gstaudiomixer gstaudiorate gstaudioresample gstaudiotestsrc
+endif
+
+# In case of Android ARM 64bit environment, the default path of linker is "/data/nnstreamer/".
+include $(CLEAR_VARS)
+LOCAL_MODULE    := nnstreamer_android
+LOCAL_CFLAGS    += -O0 -DVERSION=\"$(NNSTREAMER_VERSION)\"
```
Comment (leemgs) [inline android/nnstreamer_android/Android.mk]: Looks good to support gdb by default.

**Comment by jijoongmoon** [comment]
There is better example than this. Use it.

**Pull request closed without merging**